content/docs/user-guide/project-structure/configuration.md

```diff
@@ -394,16 +394,34 @@ have no effect.
 ## studio
 
 - `studio.token` - Studio access token to use. When this is set, DVC uses this
-  to notify Studio of new experiments. For security reasons, we advise setting
-  token to either a local or a global config. This can also be specified through
-  `DVC_STUDIO_TOKEN` environment variable.
+  to share [live metrics] and notify Studio about [pushed experiments]. For
+  security reasons, we advise setting token to either a local or a global
+  config. This can also be specified through `DVC_STUDIO_TOKEN` environment
+  variable, which will override any value in `studio.token`.
 
   [Get the token](https://studio.iterative.ai/user/_/profile?section=accessToken)
   or check
   [this guide on how to create an access token](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token).
 
+- `studio.offline` - Disables sharing [live metrics] even if `studio.token` is
```

Comment:
curious - why this name? why not "studio.share" (true/false)?

Reply:
Primarily because we will have sharing on by default. For a boolean environment variable/config option, I would rather default to false. It feels more intuitive to me and [less error-prone](https://rusingh.com/all-been-there-boolean-environment-variables/) to have the flag enable some non-default behavior (in other words, default=false).

If the default were `studio.share=true`, it feels harder for me to remember. It sounds like it needs to be enabled first, and I get confused with how to set it. I will always be worrying that by setting it to some false-y value it will end up ignored. I'd rather affirmatively set to true and be confident I am offline/not sharing.

We might change our minds on the default later, but without a strong reason to name it something else now, I'd rather go with a name that makes sense with default false.